### ISSUE TITLE
Group Lasso Math. Formulation Possible Typo

### ISSUE THREAD

**Issue by cruyffturn** [glm-tools/pyglmnet#397]
Hello,

In the [api](https://glm-tools.github.io/pyglmnet/api.html), the regularization formulation is given as below:

>   \\mathcal{P}_2 = \\|\\Gamma \\beta \\|_2^2 \\
>    \\mathcal{P}_1 = \\sum_g \\|\\beta_{j,g}\\|_2

The last summation is over g but not over j. Can it be the case that under-script "j" is a typo? I think the group lasso formulation should be the l2 norm of all coefficients in the group i.e. \\|\\beta_{g}\\|_2?

Thanks!

**Comment by jasmainak** [comment]
I agree that the formulation is strange. The coefficients should be indexed over `g` where `g \in G` or an indicator variable. Can you propose a fix with a pull request? Thank you
